.github/workflows/diff-report.yml

```diff
@@ -97,6 +97,36 @@ jobs:
           # shellcheck disable=SC2005
           echo "$(cat "$GITHUB_OUTPUT")"
 
+      - name: Extract config name
+        id: extract_config_name
+        run: |
+          CONFIG_NAME=$(grep -oP '(?<=Diff Regression config: ).*?(?=/config.xml)' .ci-temp/text | awk -F'/' '{print $(NF-1)"/"$NF}')
+          echo "config_name=$CONFIG_NAME" >> $GITHUB_OUTPUT
+          echo "Extracted config name: $CONFIG_NAME"
+
+      - name: Extract check and example
+        id: extract_check_example
+        run: |
+          CHECK_EXAMPLE=$(echo "${{ github.event.comment.body }}" | grep -oP '(?<=Github, generate report ).*')
+          echo "check_example=$CHECK_EXAMPLE" >> $GITHUB_OUTPUT
```

Comment:
when you do echo, do it for humans.
`echo "Check Example=..."`
or exactly varaible name - tracing.
` echo "CHECK_EXAMPLE=$CHECK_EXAMPLE"`